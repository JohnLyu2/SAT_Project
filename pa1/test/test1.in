
1 * VAr ;
;
1 * (VAr + (0 * b2)) ; VAr: 1, b2: 0, c:1
a b ; ab:1
1 * (-VAr + ((0+VAr) * b2)) ; VAr: 1, b2: 1, c:1
1 * (-VAr + ((0+VAr) * -b2)) ; VAr: 1, b2: 0, c:1, c:0
(1-0);
1 * (VAr + -((0 + ---c) * b2)) ; VAr: 1, b2: 0, c:1
1 * VAr + -((0 * ---c) + b2) ; VAr: 0, b2: 0, c:1
1 * VAr + -((0 * ---c) + b2) ; VAr: 0, b2: 0
