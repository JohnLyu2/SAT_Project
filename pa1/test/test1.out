Error: invalid input
Error: incomplete assignment
Error: invalid input
1
Error: invalid input
1
Error: contradicting assignment
Error: invalid input
1
1
Error: incomplete assignment
